pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1496,6 +1514,13 @@ func configToStatus(ctx *domainContext, config types.DomainConfig,
 			}
 		}
 	}
+	if need9P {
+		status.DiskStatusList = append(status.DiskStatusList, types.DiskStatus{
+			FileLocation: "/mnt",
```

Comment:
Shouldn't we add the file location of the root container disk (first disk) here?

Reply:
Not really -- but this is a subtle one (and feel free to suggest something better). We're basically introducing a fake volume here (the user didn't ask for it) the same way we introduce a fake volume when a user has passed a cloud-init (we bake an ISO volume out of cloud init on the fly).

This fake volume, however, is a proxy for ALL OCI volumes (since they will all come via a namespace of the task rather than being individually mounted inside the VM).

Makes sense?

Reply:
Makes sense. Thanks!